spec/fixtures/generate_crt_and_Key_for_1028_keylength.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir ca && cd ca
```

Comment:
Please split up these lines. && I believe will make the second command not work with if the directory creation failed; which is incorrect. If mkdir fails the whole script should error out.

Also you should just ensure the directory is made, not fail if it exists. 'mkdir -p' I think? ( check the --help )

Reply:
My intention was, if there is ca folder with some reason, I would not want to delete the folder. So I wanted to make it atomic way.